src/mish2json/Makefile

```diff
@@ -0,0 +1,2 @@
+.PHONY: all
+	@echo placeholder
```

Comment:
To follow up here: this `all` target should build the `cohort.py` parser from `cohort.ksc`.

We should also add a phony `clean` target that removes the `cohort.py`.

Reply:
i.e., something like this:

```make
.PHONY: all
all: cohort.py

cohort.py: cohort.ksc
    kaitai-struct-compiler <whatever>

.PHONY: clean
    rm -f cohort.py
```